pkg/gofr/http/multipart_file_bind.go

```diff
@@ -1,14 +1,30 @@
 package http
 
 import (
+	"encoding/json"
+	"errors"
+	"fmt"
 	"io"
 	"mime/multipart"
 	"reflect"
 	"strconv"
+	"strings"
 
 	"gofr.dev/pkg/gofr/file"
 )
 
+var (
+	errNilInterface             = errors.New("cannot set value on a nil interface")
+	errUnsupportedInterfaceType = errors.New("unsupported interface value type")
+	errDataLengthExceeds        = errors.New("data length exceeds array capacity")
+	errUnsupportedKind          = errors.New("unsupported kind")
+	errSettingValue             = errors.New("error setting value at index")
+	errNotAStruct               = errors.New("provided value is not a struct")
+	errFieldNotFound            = errors.New("field not found in struct")
+	errFieldUnexported          = errors.New("cannot set field; it might be unexported")
+	errUnsupportedFieldType     = errors.New("unsupported type for field")
```

Comment:
```suggestion
	errUnexportedField          = errors.New("cannot set field; it might be unexported")
	errUnsupportedFieldType     = errors.New("unsupported type for field")
```

Reply:
changed to **errUnexportedField**